pipelines/folioRancher/folioQualityGates/Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+import org.jenkinsci.plugins.workflow.libs.Library
+import org.folio.rest.GitHubUtility
+
+@Library('pipelines-shared-library@RANCHER-1000') _
+
+properties([
+  buildDiscarder(logRotator(numToKeepStr: '20')),
+  disableConcurrentBuilds(),
+  parameters([
+    folioParameters.agent(),
+    folioParameters.refreshParameters(),
+    ansiColor('xterm') {
+      if (params.refresh_parameters) {
+        currentBuild.result = 'ABORTED'
+        println('REFRESH JOB PARAMETERS!')
+        return
+      }
```

Comment:
Are you sure that it will work in this way?

Reply:
this property parameters we dont need here to be honest and why i insert here :(
this quality Gate  job will call(triggers)  other jobs so not need in this place like this yes you are correct,
when other stage will be implemented and if there need refresh parameters we can send as value